.github/workflows/add-stale-label.yaml

```diff
@@ -0,0 +1,35 @@
+name: Issue management - run stale action
+
+on:
+  schedule:
+    # hourly at minute 41
+    - cron: "41 * * * *"
```

Comment:
A bit specific but still works for me 😄

Reply:
https://affinitynumerology.com/number-meanings/number-41-meaning.php

Reply:
LOL I just didn't want it to be the same as the other repos, and I didn't want it to be at the top of the hour. 😁